packages/zipkin/test/batch-recorder.test.js

```diff
@@ -142,7 +142,7 @@ describe('Batch Recorder', () => {
     });
   });
 
-  it('should record minimum duration of 1 microsecond', () => {
```

Comment:
minimum duration is now checked in model.test.js as it has nothing to do with this type